### PR TITLE
[zh-cn]Sync content/zh-cn/docs/tasks/manage-gpus/scheduling-gpus.md

### PR DESCRIPTION
**Pull request by yanrongshi**
```yaml
content/zh-cn/docs/tasks/manage-gpus/scheduling-gpus.md
```

### CODE REVIEW

File: content/zh-cn/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 1]
```diff
@@ -163,24 +158,31 @@ has the following requirements:
...
 
 To deploy the NVIDIA device plugin once your cluster is running and the above
 requirements are satisfied:
 -->
 - Kubernetes 的节点必须预先安装了 NVIDIA 驱动
 - Kubernetes 的节点必须预先安装 [nvidia-docker 2.0](https://github.com/NVIDIA/nvidia-docker)
-- Docker 的[默认运行时](https://github.com/NVIDIA/k8s-device-plugin#preparing-your-gpu-nodes)必须设置为 nvidia-container-runtime，而不是 runc
-- NVIDIA 驱动版本 ~= 384.81
+- Kubelet 的容器运行时必须使用 Docker
+- Docker 的[默认运行时](https://github.com/NVIDIA/k8s-device-plugin#preparing-your-gpu-nodes)必须设置为 `nvidia-container-runtime`，而不是 runc。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/manage-gpus/scheduling-gpus.md:172]:
```suggestion
- Docker 的[默认运行时](https://github.com/NVIDIA/k8s-device-plugin#preparing-your-gpu-nodes)必须设置为
  `nvidia-container-runtime`，而不是 `runc`。
```

File: content/zh-cn/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 2]
```diff
@@ -209,13 +214,14 @@ kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/releas
...
 
 <!--
-Report issues with this device plugin and installation method to [GoogleCloudPlatform/container-engine-accelerators](https://github.com/GoogleCloudPlatform/container-engine-accelerators).
+You can report issues with using or deploying this third-party device plugin by logging an issue in
+[GoogleCloudPlatform/container-engine-accelerators](https://github.com/GoogleCloudPlatform/container-engine-accelerators).
 
 Google publishes its own [instructions](https://cloud.google.com/kubernetes-engine/docs/how-to/gpus) for using NVIDIA GPUs on GKE .
 -->
-请到 [GoogleCloudPlatform/container-engine-accelerators](https://github.com/GoogleCloudPlatform/container-engine-accelerators) 报告有关此设备插件以及安装方法的问题。
+你可以通过在 [GoogleCloudPlatform/container-engine-accelerators](https://github.com/GoogleCloudPlatform/container-engine-accelerators) 中记录问题来报告使用或部署此第三方设备插件的问题。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/manage-gpus/scheduling-gpus.md:222]: wrap the long lines where appropriate, okay?

**Review by Sea-n — COMMENTED** [review]
/approve

**Pull request merged**